README.md

```diff
@@ -12,3 +12,84 @@ Lifecycle commands:
 | `npx pnpm lint`          | lint all files                          |
 | `npx pnpm lint-staged`   | lint all staged files                   |
 | `npx pnpm test`          | run unit tests                          |
+
+## Usage
+#### CLI commands:
+Build repositories according to config:
+```sh
+jewel-case plan <repo-out> [source-dir]
```

Comment:
I think it is better to pass all the things through config. Because output directory is only applicable to static-file repos, but not for Stores (like Snapcraft).